python/covid_web/covid_web/country_dashboard.py

```diff
@@ -193,15 +194,12 @@ def update_graph(graph_axis_type_str, country_short_name):
             )
 
     def _create_single_country_all_predictions_callbacks(self, app: dash.Dash) -> None:
-        # TODO: Predictions will be generated statically in the future.
         graph_by_short_name = {}
-        for country_short_name in prediction_db.get_countries():
+        for country_short_name in self.prediction_db.get_countries():
             report = self.report_by_short_name[country_short_name]
-            country_predictions = prediction_db.predictions_for_country(country=country_short_name)
-            country_predictions.extend(
-                get_fitted_predictions(
-                    country_report=report, last_data_dates=[report.dates[-1], report.dates[-8]]
-                )
+            country_predictions = self.prediction_db.select_predictions(
+                country=country_short_name,
+                last_data_dates=[report.dates[-1], report.dates[-8]],
```

Comment:
So the `/daily` endpoint will only show two predictions, no Boďová & Kollár?

If that's the case I'd prefer also showing either `-15`. Or every 5-th day: `-11, -6, -1`.

Reply:
I think we should get rid of the endpoint now that we have the slider, what do you think?

Reply:
I actually like it, now that we've removed the BK predictions from it! It serves a different purpose than the slider, you can see more predictions at once.

But I agree that it's added complexity in the code which needs to be maintained.

Reply:
I brought the BK predictions back so that the functionality is not changed... 😇 I think I'll just commit, feel free to change the functionality again.